test/support/markdown_editor_helpers.rb

```diff
@@ -1,5 +1,13 @@
 module MarkdownEditorHelpers
   def fill_in_editor(text)
+    sleep(0.5)
     execute_script("document.querySelector('.CodeMirror').CodeMirror.setValue('#{text}')")
   end
+
+  def assert_editor_text(expected)
+    sleep(0.5)
```

Comment:
Maybe add a comment why this is needed?

Reply:
I've wrapped this in a method named `wait_for_editor_to_load`.